tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -488,6 +488,37 @@ describe(__filename, () => {
     );
   });
 
+  it('shows tag links if addon tags defined', () => {
+    const addon = createInternalAddonWithLang(fakeAddon);
```

Comment:
It is a bit better to declare the tags here instead of relying on default values that might change.